Api/HttpContextClaimsProvider.cs

```diff
@@ -21,4 +24,12 @@ public long TenantId
             return _tenantId;
         }
     }
+
+    public long EmployeeId
```

Comment:
nit: Expression body properties can be used here for brevity:
`public long EmployeeId => _employeeId;`